plugins/smb/komand_smb/actions/create_file/action.py

```diff
@@ -0,0 +1,69 @@
+import komand
+from .schema import CreateFileInput, CreateFileOutput, Input, Output, Component
+from komand.exceptions import PluginException
+
+# Custom imports below
+import io
+import smb
+import base64
+
+
+class CreateFile(komand.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="create_file", description=Component.DESCRIPTION, input=CreateFileInput(), output=CreateFileOutput()
+        )
+
+    def run(self, params={}):
+        file_path = params.get(Input.FILE_PATH)
+        share_name = params.get(Input.SHARE_NAME)
+        file_content = base64.b64decode(params.get(Input.FILE_CONTENT))
```

Comment:
```suggestion
        file_content = params.get(Input.FILE_CONTENT)
        try:
            file_content = base64.b64decode(file_content)
        except binascii.Error as e:
            raise PluginException(cause="Invalid file content input provided.", assistance="Please double-check the file content input.")
```

It looks like it's possible that b64decode can throw an error when the input type is just a standard string.